_includes/sidebar.html

```diff
@@ -123,7 +123,7 @@
   </li>
   {% endif %} {% endfor %}
   <div class="sidebarbuttoncontainer">
-    <li class="folder topfolder">
+    <div class="folder topfolder">
```

Comment:
I can follow your thinking, but here it may be okay to instead make the `sidebarbuttoncontainer` a list. Reasoning: It could contain more items than just KC or DS.

Optional: Maybe move the DS link into this list, too? Then we kill two birds with one stone.

Reply:
good idea, done